README.md

```diff
@@ -462,32 +464,89 @@ values stored in `argv._`).  Set `desc` to `false` to create a hidden command.
 Hidden commands don't show up in the help output and aren't available for
 completion.
 
-Optionally, you can provide a handler `fn` which will be executed when
-a given command is provided. The handler will be called with `yargs` and
-`argv` as arguments.
-
-`yargs` is a blank instance of yargs, which can be used to compose a nested
-hierarchy of options handlers.
+Optionally, you can provide a `builder` object to give hints about the
+options that your command accepts:
 
-`argv` represents the arguments parsed prior to the
-command being executed (those described in the outer yargs instance).
+```js
+yargs.command('get', 'make a get HTTP request', {
+    url: {
+      default: 'http://yargs.js.org/'
+    }
+  })
+  .help()
+  .argv
+```
```

Comment:
@lrlna I've updated the documentation for **command modules** which I think are the most elegant approach we can direct people towards.